sdk/communication/Azure.Communication.JobRouter/README.md

```diff
@@ -0,0 +1,298 @@
+# Azure Communication JobRouter client library for .NET
+
+This package contains a C# SDK for Azure Communication Services for JobRouter.
+
+[Source code][source] | [Package (NuGet)]<!-- [https://www.nuget.org/packages/Azure.Communication.JobRouter]--> | [Product documentation][product_docs]
+
+
+## Getting started
+
+### Install the package
+Install the Azure Communication JobRouter client library for .NET with [NuGet][nuget]:
+
+```dotnetcli
+dotnet add package Azure.Communication.JobRouter --prerelease
+``` 
+
+### Prerequisites
+You need an [Azure subscription][azure_sub] and a [Communication Service Resource][communication_resource_docs] to use this package.
+
+To create a new Communication Service, you can use the [Azure Portal][communication_resource_create_portal], the [Azure PowerShell][communication_resource_create_power_shell], or the [.NET management client library][communication_resource_create_net].
+
+### Authenticate the client
+
+### Using statements
+```C# Snippet:Azure_Communication_JobRouter_Tests_Samples_UsingStatements
+using Azure.Communication.JobRouter;
+```
+
+### Create a JobRouter Client
+
+This will allow you to interact with the JobRouter Service
+```C#
```

Comment:
This should be a snippet.

Reply:
This has been updated to be injected with snippet